crds/AzureApplicationGatewayResponse_v1beta1.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: apiextensions.k8s.io/v1beta1 
+kind: CustomResourceDefinition 
+metadata: 
+  name: azureapplicationgatewayresponses.appgw.ingress.azure.io 
```

Comment:
How about `azureapplicationgatewayinstancestatus`?

Reply:
Updated to azureapplicationgatewayinstanceupdatestatus